# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\xampp\htdocs\siteBuilder2\sureFramework\trunk\server\www\javascript\jquery
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: jquery-ui-1.7.2.custom.js
--- jquery-ui-1.7.2.custom.js Base (BASE)
+++ jquery-ui-1.7.2.custom.js Locally Modified (Based On LOCAL)
@@ -4382,7 +4382,7 @@
 			.filter(':first')
 			.focus();
 
-		this._trigger('open');
+		this._trigger('open', null, uiDialog);
 		this._isOpen = true;
 	},
 
